Example/RxAppState_ExampleTests/RxAppStateTests.swift

```diff
@@ -118,6 +120,119 @@ class RxAppStateTests: XCTestCase {
         XCTAssertEqual(firstLaunchArray, [true])
     }
     
+    func testisFirstLaunchOfNewVersionNewInstall() {
```

Comment:
This is a really small unimportant detail, but would you mind using a capital letter after the `test` in your test method names? Just to have consistency with the existing test method names. So in this case it would be `testIsFirstLaunchOfNewVersionNewInstall()`